packages/language/src/index.js

```diff
@@ -0,0 +1,432 @@
+ * function callback(err, entities, apiResponse) {}
+ *
+ * language.detectEntities('Axel Foley is from Detroit', callback);
+ *
+ * //-
+ * // Or, provide a reference to a file hosted on Google Cloud Storage.
+ * //-
+ * var gcs = gcloud.storage();
+ * var bucket = gcs.bucket('my-bucket');
+ * var file = bucket.file('my-file');
+ *
+ * language.detectEntities(file, callback);
+ *
+ * //-
+ * // Specify HTML content.
+ * //-
+ * var options = {
+ *   type: 'html'
+ * };
+ *
+ * language.detectEntities('Axel Foley is from Detroit', options, callback);
+ *
+ * //-
+ * // Verbose mode may also be enabled for more detailed results.
+ * //-
+ * var options = {
+ *   verbose: true
+ * };
+ *
+ * language.detectEntities('Axel Foley is from Detroit', options, callback);
+ */
+Language.prototype.detectEntities = function(content, options, callback) {
+  if (is.fn(options)) {
+    callback = options;
+    options = {};
+  }
+
+  options = extend({}, options, {
+    content: content
+  });
+
+  var document = this.document(options);
+  document.detectEntities(options, callback);
+};
+
+/**
+ * Detect the sentiment of a block of text.
+ *
+ * NOTE: This is a convenience method which doesn't require creating a
+ * {module:language/document} object. If you are only running a single
+ * detection, this may be more convenient. However, if you plan to run multiple
+ * detections, it's easier to create a {module:language/document} object.
+ *
+ * @resource [documents.analyzeSentiment API Documentation]{@link https://cloud.google.com/natural-language/reference/rest/v1beta1/documents/analyzeSentiment}
+ *
+ * @param {string|module:storage/file} content - Inline content or a Storage
+ *     File object.
+ * @param {object=} options - Configuration object.
+ * @param {string} options.encoding - `UTF8`, `UTF16`, or `UTF32`. See
+ *     [`EncodingType`](https://cloud.google.com/natural-language/reference/rest/v1beta1/EncodingType).
+ * @param {string} options.language - The language of the text.
+ * @param {string} options.type - The type of document, either `html` or `text`.
+ * @param {boolean} options.verbose - Enable verbose mode for more detailed
+ *     results. Default: `false`
+ * @param {function} callback - See {module:language/document#detectSentiment}.
+ *
+ * @example
+ * //-
+ * // See {module:language/document#detectSentiment} for a detailed breakdown of
+ * // the arguments your callback will be executed with.
+ * //-
+ * function callback(err, sentiment, apiResponse) {}
+ *
+ * language.detectSentiment('Hello!', callback);
+ *
+ * //-
+ * // Or, provide a reference to a file hosted on Google Cloud Storage.
+ * //-
+ * var gcs = gcloud.storage();
+ * var bucket = gcs.bucket('my-bucket');
+ * var file = bucket.file('my-file');
+ *
+ * language.detectSentiment(file, callback);
+ *
+ * //-
+ * // Specify HTML content.
+ * //-
+ * var options = {
+ *   type: 'html'
+ * };
+ *
+ * language.detectSentiment('&lt;h1>Document Title&lt;/h1>', options, callback);
```

Comment:
Nevermind, just noticed your docs render correctly like this.. kind of a weird quirk!